.github/workflows/build.yaml

```diff
@@ -6,7 +6,7 @@ on:
 
 jobs:
     check:
-        runs-on: ubuntu-20.04
+        runs-on: ubuntu-22.04
```

Comment:
@garak Are you fine with changing it to `ubuntu-latest` (for all cases)?

Reply:
I would avoid "latest" because there's no guarantee that the next version of Ubuntu will work as the current one.
Updating it all the time can be tedious, but it's explicit.

Reply:
I am not sure what to do. I stick now with 22.04. Maybe make a seperat PR and discuss that.